e2e/parallel/nftsFlow.test.ts

```diff
@@ -0,0 +1,203 @@
+/* eslint-disable no-await-in-loop */
+import 'chromedriver';
+import 'geckodriver';
+import { WebDriver } from 'selenium-webdriver';
+import { afterAll, beforeAll, describe, expect, it } from 'vitest';
+
+import {
+  delayTime,
+  findElementByTestId,
+  findElementByTestIdAndClick,
+  findElementByText,
+  getExtensionIdByName,
+  getRootUrl,
+  goToPopup,
+  importWalletFlow,
+  initDriverWithOptions,
+  toggleStatus,
+} from '../helpers';
+import { TEST_VARIABLES } from '../walletVariables';
+
+let rootURL = getRootUrl();
+let driver: WebDriver;
+
+const browser = process.env.BROWSER || 'chrome';
+const os = process.env.OS || 'mac';
+
+describe('Navigate Settings & Privacy and its flows', () => {
+  beforeAll(async () => {
+    driver = await initDriverWithOptions({
+      browser,
+      os,
+    });
+    const extensionId = await getExtensionIdByName(driver, 'Rainbow');
+    if (!extensionId) throw new Error('Extension not found');
+    rootURL += extensionId;
+  });
+  afterAll(async () => await driver.quit());
+
+  it('should be able import a wallet via seed', async () => {
+    await importWalletFlow(driver, rootURL, TEST_VARIABLES.EMPTY_WALLET.SECRET);
+  });
+
+  it('should be able to navigate to the nfts tab (prerelease)', async () => {
+    await goToPopup(driver, rootURL);
+    await findElementByTestIdAndClick({ id: 'bottom-tab-nfts', driver });
+    const nftPrereleaseText = await findElementByText(
+      driver,
+      'NFTs are coming soon. Stay tuned!',
+    );
+    expect(nftPrereleaseText).toBeTruthy();
+  });
+
+  it('should be able to enable the nft feature flag', async () => {
+    await findElementByTestIdAndClick({ id: 'home-page-header-right', driver });
+    await findElementByTestIdAndClick({ id: 'settings-link', driver });
+    await findElementByTestIdAndClick({
+      id: 'feature-flag-nfts_enabled',
+      driver,
+    });
+    expect(await toggleStatus('feature-flag-nfts_enabled', driver)).toBe(
+      'true',
+    );
+  });
+
+  it('should be able to return nfts tab and view nfts', async () => {
+    await findElementByTestIdAndClick({
+      id: 'navbar-button-with-back',
+      driver,
+    });
+    await findElementByTestIdAndClick({ id: 'bottom-tab-nfts', driver });
+  });
+
+  it('should be able to navigate to NFT details', async () => {
+    await delayTime('short');
+    await findElementByTestIdAndClick({
+      id: 'nft-thumbnail-https://lh3.googleusercontent.com/ICZJsSKEfLTquCvn1o-W7wD75EdtCksjf3bMtm2IQsYdw7K8-_de9gGQBXJE09fHy33OtBBrgWqMUAfX2ve6ZsW200JnLrX-m3s=s1000-0',
```

Comment:
is there a way to use the nft id or something else? this looks fragile if the metadata gets updated somehow

Reply:
The data for the 2 test NFTs is a faked constant, so this string won't change unless we do it directly.

I could change it to be more readable if you want, but I was hesitant to add props to the component just for testing purposes.